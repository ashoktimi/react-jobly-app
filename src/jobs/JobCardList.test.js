import React from 'react';
import { render } from '@testing-library/react';
import JobCardList from './JobCardList';

describe('JobCardList', () => {
  it('matches snapshot', () => {
    const { container } = render(<JobCardList jobs={[]} apply={() => {}} />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    const { container } = render(<JobCardList jobs={[]} apply={() => {}} />);
    expect(container.firstChild).toBeInTheDocument();
  });
});