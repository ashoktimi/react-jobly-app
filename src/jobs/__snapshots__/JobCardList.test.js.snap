// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobCardList matches snapshot 1`] = `
<div
  class="JobCardList"
/>
`;
